adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -155,6 +157,25 @@ public ITokenCacheStore getCache() {
     }
 
     /**
+     * Gets if the outage mode is enabled
+     *
+     * @return True when outage mode is enabled
+     */
+    public boolean getExtendedLifetimeEnabled() {
+        return mExtendedLifetimeEnabled;
+    }
+
+    /**
+     * The client need to set the extendedLifetimeEnabled explicitly to enable the outage mode
+     * The default value of mExtendedLifetimeEnabled is false.
```

Comment:
This is a public interface, I don't think we should say mExtendedLifetimeEnabled in the comment, it's our internal implementation. :)
Instead I think we should say more i.e. what does the flag do, and under which case you get the stale token.